prow/pod-utils/decorate/podspec.go

```diff
@@ -593,6 +594,15 @@ func decorate(spec *coreapi.PodSpec, pj *prowapi.ProwJob, rawEnv map[string]stri
 	// TODO(fejta): we should pass around volume names rather than forcing particular mount paths.
 
 	rawEnv[artifactsEnv] = artifactsPath
+	gcsConfig := pj.Spec.DecorationConfig.GCSConfiguration
+	rawEnv[gcsArtifactsDirEnv] = path.Join(
+		gcsConfig.Prefix,
```

Comment:
I do not see Prefix here:
https://github.com/kubernetes/test-infra/blob/master/prow/apis/prowjobs/v1/types.go#L535

Reply:
Outdated - `PathPrefix` is handled inside `PathsForJob` call. Good catch, though.